extensions/mongodb/src/main/java/com/hazelcast/jet/sql/impl/connector/mongodb/ConversionsFromBson.java

```diff
@@ -57,7 +57,7 @@ private ConversionsFromBson() {
      * In other words, converts for SELECT statements.
      */
     @Nullable
-    @SuppressWarnings("checkstyle:RightCurly")
+    @SuppressWarnings({ "checkstyle:RightCurly", "javabugs:S6320" })
```

Comment:
Could you add a comment what the `S6320` means?

Reply:
Removed completely after small refactor